DSCResources/xTimeZone/xTimeZone.psm1

```diff
@@ -13,7 +13,7 @@ function Get-TargetResource
     )
 
     #Get the current TimeZone
-    $CurrentTimeZone = Invoke-Expression "tzutil.exe /g"

```

Comment:
more specifically, why did line 16, and line 40 change?  there must be some sort of white space change, can you please explain?